manager/runner.go

```diff
@@ -309,6 +310,13 @@ func (r *Runner) Start() {
 
 		if r.allTemplatesRendered() {
 			log.Printf("[DEBUG] (runner) all templates rendered")
+
+			// Send a signal to systemd that we are ready.
+			_, err := daemon.SdNotify(false, daemon.SdNotifyReady)
```

Comment:
This is going to be problematic for downstream applications that consume consul-template as a library, such as Nomad. This would cause Nomad to fire a readiness signal every time an allocation has its templates rendered, which we don't want here.

We should send the notification signal from a goroutine on `main` and then send signals from here to that goroutine when it's available.

Reply:
Hello @tgross 
I completely agree with you. I have changed the approach to implementation according to your recommendations.
Thx